### ISSUE TITLE
Vulnerator will not launch in Win 10 x64 build 1607 LTSB

### ISSUE THREAD

**Issue by PCGOOD** [Vulnerator/Vulnerator#152]
#### Prerequisites
_Before submitting a new issue, please ensure you have completed the following (replace the space in the box with an "x" to denote that it has been completed)_

- [x] I have ensured that I am running the latest release
- [x] The issue is repeatable
- [ ] The issue has not already been reported
- [ ] The issue does not fit into one of the suggested categories (Assistance Request, Bug Report, or Feature Request)

#### Description
Vulnerator will not launch in Windows 10 (1607 LTSB build). 
I have the full extraction of the .zip file to c:\vulnerator folder.  Double click to launch, does not produce Event Log in Windows, Does not produce log file on Desktop or in root folder. Process launches and immediately stops. No entry in HIPs logs.  Vulnerator log not produced in ```C:\Users\%UserName%\AppData\Roaming\Vulnerator\V6Log.txt``` 

Windows 10 x64 bit version 1607 LTSB
.NET Framework 4.6.2
Full DoD STIG system
User Rights (Admin privileges not available)
Windows UAC at full setting

#### System Information
 -OS: Windows 10 x64
 -Version: 10.0.14393

#### Additional Context
_Add any other context about the problem here, if applicable._

#### Attachments
_Please provide any relevant attachments, as you see fit (e.g. screenshots); if supplying vulnerability data (e.g. CKL/Nessus files or reports), please ensure that they are sanitized of IP addresses and host names and **email** them to alex.kuchta@navy.mil - **DO NOT POST VULNERABILITY FILES HERE**_

- [ ] Vulnerator Log (required for bugs) - this can be found at ```C:\Users\%UserName%\AppData\Roaming\Vulnerator\V6Log.txt```, where ```%UserName%``` is the user profile used to run the application
- [ ] Screenshots (optional, _**SANITIZED/REDACTED**_)
- [ ] Vulnerability Files (optional, _**SANITIZED AND EMAILED**_)

**Comment by amkuchta** [comment]
@PCGOOD you referenced the log file path - did you replace `%UserName%` with your username (I don't want to insult your intelligence, but I have to ask)

**Comment by PCGOOD** [comment]
Yes,  I looked for the log file in the folder directory with my User Name instead of %UserName%.

No log file is being created when you attempt to launch vulnerator.  We do not use NMCI computers.  Ours are provided by our Project and the systems are fully STIG'd, so this might be why the file cannot create.

Question: on first run of Vulnerator, will it attempt to store or write files to any of the system TEMP/TMP folders?  Reason being, is the rules for HBSS prevent any executable file from running in a TEMP/TMP folder.   I have seen other custom applications that had to be modified to use their installation directory for temp file execution to bypass this STIG.

**Comment by PCGOOD** [comment]
Please send email to paul.good.ctr@navy.mil

**Comment by PCGOOD** [comment]
Alex,  I have not received an email with the Army SAFE link yet?  Hope you did not forget.

**Comment by amkuchta** [comment]
@PCGOOD I'm surprised it did not go through - I will resend tomorrow when I am back in the office.  Sorry for the late response - I started a new job last week, and I have been working on getting my feet under me.

**Comment by sstretchh** [comment]
I am also having the same issue running in windows 10

**Comment by amkuchta** [comment]
@sstretchh thank you for the amazing amount of information!  Ok, lets start by completing the following:

1. Ensure that all instances of Vulnerator are killed within the Task Manager
2. Navigate to the below directory and delete the "Vulnerator" folder
    - `C:/Users/Your.Account/AppData/Roaming`
    - This is a hidden folder, so you may need to unhide it
3. Re-run Vulnerator
4. Navigate to the below folder and report back what files exist in the directory
    - `C:/Users/Your.Account/AppData/Roaming/Vulnerator`

**Comment by backrhoades** [comment]
> @sstretchh thank you for the amazing amount of information!  Ok, lets start by completing the following (@eng3 @BillCosner you may want to try these as well):
> 
> Ensure that all instances of Vulnerator are killed within the Task Manager
> Navigate to the below directory and delete the "Vulnerator" folder
> 
> C:/Users/Your.Account/AppData/Roaming
> This is a hidden folder, so you may need to unhide it
> 
> 
> Re-run Vulnerator
> Navigate to the below folder and report back what files exist in the directory
> 
> C:/Users/Your.Account/AppData/Roaming/Vulnerator

I followed this and it fixed it. Mahalo!

**Comment by eng3** [comment]
This is a different issue.  The issue I am having is that my users' AppData/Roaming folder is on a network drive.  When using Folder Redirection or Roaming Profiles, Vulnerator seems to have issues with accessing AppData/Roaming/Vulnerator files.  If I turn off Folder Redirection/Roaming Profiles, it works fine.
